src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/ValidationRulePart.cshtml

```diff
@@ -0,0 +1,58 @@
+@using OrchardCore.Forms.Models
+@model ShapeViewModel<ValidationRulePart>
+@{
+    var type = Model.Value.Type;
+    var option = Html.Raw(Model.Value.Option);
+    var formElementPart = Model.Value.ContentItem.As<FormElementPart>();
+    var elementId = formElementPart.Id;
+}
+
+<script asp-src="~/OrchardCore.Forms/Scripts/validator.min.js" debug-src="~/OrchardCore.Forms/Scripts/validator.js" asp-name="validator" depends-on="jQuery" at="Foot"></script>
+
+<script at="Foot">
+    function validateElementFunction(type, originElementId, option) {
```

Comment:
Client-side validation errors will only appear if you've previously edited the given field:

https://user-images.githubusercontent.com/1976647/114029067-71a47480-9879-11eb-8c69-d2772d10be4c.mp4

I think they should appear when you try to submit the form in also if you didn't edit them.